homeassistant/components/input_datetime/services.yaml

```diff
@@ -0,0 +1,9 @@
+set_datetime:
+  description: This can be used to dynamically set the date and/or time.
+  fields:
+    entity_id: {description: Entity id of the input datetime to set the new value.,
```

Comment:
Why are you using such weird YAML constructs. Sure it's valid, but does not look nice?

Reply:
I just copied from others, for example https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/input_select/services.yaml.
Not correct?

Reply:
Correct, but could be better 😉 

I'll see if I can write a script to fix them all.